packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerSection.tsx

```diff
@@ -9,4 +10,25 @@ const StyledSection = styled.div`
   flex-shrink: 1;
 `;
 
-export { StyledSection as NavigationDrawerSection };
+const StyledSectionInnerContainerMinusScrollPadding = styled.div<{
+  isMobile: boolean;
+}>`
+  width: calc(
+    100% - ${({ isMobile, theme }) => (isMobile ? 0 : theme.spacing(2))}
+  );
+`;
+
+export const NavigationDrawerSection = ({
```

Comment:
I don't understand all these changes? Looks big changes for a small fix, could you explain it a bit more?

Reply:
We can't just remove scrollHide and use the ScrollWrapper in the drawer since it overlaps over the NavigationDrawerItems(for reference #9026), so the idea is to remove the right padding of the NavigationDrawers animated container. And since NavigationDrawerSection controls the width, have an internal wrapper that would minus the width required for the scroll.
Is this the right approach? also, ignore the scroll paddings and widths being changed in ScrollWrapper, I was just trying something out!